radio/src/pulses/multi_arm.cpp

```diff
@@ -46,7 +46,7 @@ static void sendByteMulti(uint8_t b)
     parity = parity ^ (b & 1);
     b >>= 1;
   }
-  putDsm2SerialBit(parity);      // Even Parity bit
+  putDsm2SerialBit(!parity);     // Even Parity bit
```

Comment:
I am not sure about this change. Wouldn't that change the parity on the other platforms as well?